### PR TITLE
Add pods package

### PR DESCRIPTION
**Pull request by wwwil**
This PR moves the previously private `pods` package to be open source, and updates the rule ids to use the new convention.

### CODE REVIEW

**Comment by charlieegan3** [comment]
I'd have this replace https://github.com/jetstack/preflight/tree/master/preflight-packages/examples.jetstack.io/pods_basic - personally.

**Comment by wwwil** [comment]
> I'd have this replace https://github.com/jetstack/preflight/tree/master/preflight-packages/examples.jetstack.io/pods_basic - personally.

I've removed the `pods_basic` package, intended to do that but forgot. I think as this is a full package rather than a reduced version of a private package it should have a different namespace though.

File: preflight-packages/jetstack.io/pods/pods.rego [hunk 1]
```diff
@@ -0,0 +1,188 @@
+package preflight.pods
```
Comment (wwwil) [inline preflight-packages/jetstack.io/pods/pods.rego]: Should we still have the `preflight` prefix in the Rego package name?

Comment (charlieegan3) [reply preflight-packages/jetstack.io/pods/pods.rego]: Do we need it? If not my vote would be to remove - but we still have it in our other policies... might be better to remove it all at once - over all packages.

**Comment by charlieegan3** [comment]
> > I'd have this replace https://github.com/jetstack/preflight/tree/master/preflight-packages/examples.jetstack.io/pods_basic - personally.
> 
> I've removed the `pods_basic` package, intended to do that but forgot. I think as this is a full package rather than a reduced version of a private package it should have a different namespace though.

Yup, agreed, it doesn't make sense to call this pods_basic any more.

File: preflight-packages/jetstack.io/pods/pods.rego (hunk 1 above)
Comment (j-fuentes) [reply preflight-packages/jetstack.io/pods/pods.rego]: it is not needed, just drop it from the `package` statement in the rego and from the `root-query` in the policy manifest.

Comment (wwwil) [reply preflight-packages/jetstack.io/pods/pods.rego]: Done!

**Pull request merged**